src/app-pages/map/MapPage.js

```diff
@@ -8,11 +8,13 @@ import Map from "./components/map/Map";
 const MapPage = ({
   queryObject,
   doLocationDetailFetch,
+  doLocationLevelFetch,
 }) => {
   const locationId = queryObject.locationId || "";
   useEffect(() => {
     doLocationDetailFetch();
-  }, [locationId, doLocationDetailFetch])
+    doLocationLevelFetch();
```

Comment:
Should we move the logic to load the location detail, level, and probably other calls that will be needed in the future, into the MapDetailsContainer? Seems like the responsibility for loading the details would be better placed in the detail container. Thoughts?

Reply:
yea makes sense, pushed up a change for that